### ISSUE TITLE
Custom mountpoints aren't preserved

### ISSUE THREAD

**Issue by Mrdini** [Ornias1993/jailman#90]
Following on from discussions the other day on the FreeNAS forums...

Any custom mountpoints should be preserved if doing a `-r` reinstall (maybe by parsing `iocage fstab JAIL -l`?)

**Comment by Ornias1993** [comment]
I'll look into creating settings for adding additional media mountpoints, but I'm not much a fan of parsing FreeNAS. It inevitably leads to (user) error.

**Comment by Mrdini** [comment]
Sure - maybe there's a better way, I'm by no means an expert on FreeNAS/jails!

Its not just media necessarily - this feature may be handy for other blueprints where users may want to add extra mountpoints/datasets.

**Comment by Ornias1993** [comment]
The thing is: This script is not a replacement for custom jails.
It's a tool for people wanting slightly more control than traditional plugins and a slightly easier reinstall process.

Creating our own form of fstab ( or parsing the one already in freenas) is quite out-of-scope.

**Comment by Ornias1993** [comment]
Okey I've looked into your proposal(s) @Mrdini 

Current design proposal:
- Follow Docker Compose's "volume" syntax (origin/target:settings1:setting2 ) seems a good way of handling it.
- Move all folder creation and mounting to the jail creation (from install)
- Have the option to dictate them both in blueprint and jail config
- When in conflict: Jail config takes precedence but will not be supported (use on your own risk)

**Comment by Ornias1993** [comment]
Current design is blocked by #104

**Comment by Ornias1993** [comment]
Due to blocking, moved to 1.4
